src/tools/info/version_info.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2022, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#ifdef HAVE_CONFIG_H
+#  include "config.h"
+#endif
+#include "ucx_info.h"
+
+#include <ucs/sys/lib.h>
+#include <ucs/sys/string.h>
+
+#include <libgen.h>
+
+void print_version()
+{
+    const char *path_p;
+    char path[PATH_MAX];
+
+    printf("# Library version: %s\n", ucp_get_version_string());
+    path_p = ucs_sys_get_lib_path();
+    if (path_p != NULL) {
+        ucs_strncpy_safe(path, path_p, PATH_MAX);
+        printf("# Library path: %s\n", dirname(path));
+    } else {
+        printf("# Cannot get path to library\n");
```

Comment:
1. better print full path, no need to remove file name, in case multiple SO files/symlink/etc
2. better keep "Library path" title in both cases to make the output more consistent
```c
path = ucs_sys_get_lib_path();
printf("# Library path: %s\n", (path != NULL) ? path : "<unknown>");
```

Reply:
Done, but doesn't it feels strange that we prints only path to UCS library in this case. WDYTA printing path to UCT/UCP too (requires introducing additional APIs)?

Reply:
maybe add helper func to ucs like `const char *ucs_get_dl_path(void *func)` to get library path by symbol name?
use it from ucs_sys_get_lib_path and also from ucp_init_version, and ucx_info tool (with UCP symbols)